src/main/java/uk/co/ramp/people/PopulationGenerator.java

```diff
@@ -21,14 +19,22 @@
 @Service
 public class PopulationGenerator {
 
-    private final StandardProperties runProperties;
-    private final PopulationProperties properties;
-    RandomDataGenerator dataGenerator;
+    private StandardProperties runProperties;
+    private PopulationProperties properties;
+    private RandomDataGenerator dataGenerator;
 
     @Autowired
-    public PopulationGenerator(final StandardProperties runProperties, final PopulationProperties properties, final RandomDataGenerator dataGenerator) {
```

Comment:
I believe injecting via constructor is generally preferred over setter injection.

https://stackoverflow.com/questions/21218868/explain-why-constructor-inject-is-better-than-other-options
https://spring.io/blog/2007/07/11/setter-injection-versus-constructor-injection-and-the-use-of-required/
https://www.javacodegeeks.com/2019/02/field-setter-constructor-injection.html

Reply:
Done